lib/bolt/config.rb

```diff
@@ -110,7 +110,11 @@ def normalize_interpreters(interpreters)
     def normalize_log(target)
       return target if target == 'console'
       target = target[5..-1] if target.start_with?('file:')
-      'file:' + File.expand_path(target)
+      if @future
```

Comment:
I think normalize_log, ends up being called before @future is defined https://github.com/puppetlabs/bolt/pull/1364/files#diff-9637f8192166af0313bc199f0706d20bR140 . So @future will always be nil and this will never be relative to boltdir.

Reply:
I think evaluating future in the first line of update_from_file is safe

Reply:
Actually tested it this time :sweat_smile:

Reply:
yeah, its always the seeminly simplest changes that dont end up working like you expect! Thanks.